src/test/java/com/deloitte/ddwatch/services/QualityReportServiceTest.java

```diff
@@ -32,13 +39,110 @@ public void setup() {
     public void createSonarQubeReport() {
```

Comment:
I know this wasn't added by you @maria-neagu, but please give it a proper name 😄